packages/cactus-common/src/test/typescript/integration/integration.test.ts

```diff
@@ -0,0 +1,23 @@
+// tslint:disable-next-line: no-var-requires
```

Comment:
@gabrielsoaresm94 Did you add this file back intentionally or is it just a rebase gone bad? (my commit removed it because it was redundant)

Reply:
@petermetz I did not merge automatically, because of that some files remained, but I finished correcting it.